### PR TITLE
Fixes:[T348600] Title not being shown and placeholder text is getting cut

### PR DESCRIPTION
**Pull request by okerekechinweotito**
Fixes: [T348600](https://phabricator.wikimedia.org/T348600)

## Proposed Changes
- The title/identifier should be displayed properly.
- The placeholder text should say "Enter valid identifier"
- The validation should accept either Alphabets only or Alphanumeric characters
- Restructure and Move Identifier validation into server.js

## Files Created/Updated

- components/Books.js - receive isInValidIdentifier response from server and implement frontend. Update the changeIdentifier placeholder
- server.js - validate identifier and return isInValidIdentifier: true if validation fails

## Current Look
Screenshot (40)

## Checklist 
- [x] Coding Conventions are followed.
- [x] Comments are used for Documenting the Code.
- [x] Correct File Names are mentioned.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -387,7 +387,13 @@ app
                 IAtitle.trim() !== ""
                   ? replaceTitle(IAtitle.trim())
                   : replaceTitle(data.volumeInfo.title);
-              if ((await checkIfFileExistsAtIA(titleInIA)) === true) {
+              const isAlphanumericLess50 = /^(?![0-9]+$)[a-zA-Z0-9]{1,50}$/;
```
Comment (coderwassananmol) [inline server.js:390]: @okerekechinweotito This is failing with only numbers and no alphabet. Only number(s) is a valid identifier.

File: server.js [hunk 2]
```diff
@@ -426,11 +432,17 @@ app
             IAtitle.trim() !== ""
               ? replaceTitle(IAtitle.trim())
               : replaceTitle(await getPDLTitle(options));
+          const isAlphanumericLess50 = /^(?![0-9]+$)[a-zA-Z0-9]{1,50}$/;
```
Comment (coderwassananmol) [inline server.js:435]: Please define regex once and use it everywhere. Don't define it multiple times.

**Review by coderwassananmol — CHANGES_REQUESTED** [review]
Please fix the changes.

File: server.js (hunk 1 above)
Comment (okerekechinweotito) [reply server.js]: @coderwassananmol  fixed

File: server.js (hunk 2 above)
Comment (okerekechinweotito) [reply server.js]: @coderwassananmol  fixed

**Comment by okerekechinweotito** [comment]
@coderwassananmol  Bugs fixed

**Review by coderwassananmol — APPROVED** [review]

**Pull request merged**